README.md

```diff
@@ -59,30 +72,32 @@ We don't advise you to download a local version without using the NPM package :
 
 Initialize the lib with the Yelda provided elements
 
-- assistantSlug : the webchat short name on Yelda
-- assistantId : the webchat id on Yelda
-- assistantUrl : `https://app.yelda.ai`
-- chatPath : `/chat`
-- locale : `fr_FR`
-- isAdmin : false (set to true only for intent analysis)
-- shouldBeOpened : false
-- parentContainerId : `container_id` DOM element id where you want to add the iframe. If the 'container_id' does not exist in the DOM, the iframe will be automatically inserted into the document body
-- isStartBtn: false (set to true only if need to show start button)
-- canBeClosed: true (used to toggle the visible state of close button, if parentContainerId provided 'canBeClosed' set to false)
+| Name              | Type    | Default                | Description                                                                                                                                                        |
+| ----------------- | ------- | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| assistantSlug     | String  |                        | The assistant short name on Yelda **Required**                                                                                                                     |
+| assistantId       | String  |                        | The assistant id on Yelda **Required**                                                                                                                             |
+| assistantUrl      | String  | `https://app.yelda.ai` |
+| chatPath          | String  | `/chat`                |
+| locale            | String  | `fr_FR`                |
+| isAdmin           | Boolean | `false`                | Set to true only to access the conversations analysis                                                                                                              |
+| shouldBeOpened    | Boolean | `false`                |
+| parentContainerId | String  |                        | DOM element id where you want to add the iframe. If the 'container_id' does not exist in the DOM, the iframe will be automatically inserted into the document body |
+| isStartBtn        | Boolean | `false`                | Set to true only if need to show 'start' button                                                                                                                    |
```

Comment:
Set to true only if need to show 'start' button 

isAdmin feature (if true but isAdmin is false, won't display the startButton)